Makefile

```diff
@@ -1,10 +1,24 @@
-.PHONY: generate localmkdocs test
+.PHONY: generate regenerate localmkdocs test
 
 LATEST := v10.0.0
```

Comment:
Latest would be 10.1, no?

Reply:
Working on that one here: https://github.com/grafana/grok/pull/34
Will do that in a separate PR.